docker/default_image/Dockerfile.cpu

```diff
@@ -0,0 +1,123 @@
+FROM ubuntu:xenial
+MAINTAINER CodaLab Team "codalab.worksheets@gmail.com"
+
+# Installed packages:
+
+# default-jdk
```

Comment:
Is this going to give you Oracle (would be ideal)?

Reply:
Not sure, probably a better idea to specify Oracle directly.